cloudformation/tests/test_cfn_template_rendering.py

```diff
@@ -199,7 +199,7 @@ def test_hit_substack_rendering(tmp_path, test_config):
             "vol1,vol2,vol3,vol4,vol5",
             "gp2,io1,sc1,st1,gp2",  # No io1 metrics
             "raid,1,2,gp2,20,100,false,NONE",
-            "/fsx,NONE,1200,NONE,NONE,NONE,NONE,NONE,NONE,NONE,NONE,NONE,NONE,NONE",
+            "/fsx,NONE,1200,NONE,NONE,NONE,NONE,NONE,NONE,NONE,NONE,NONE,NONE,NONE,NONE,NONE,NONE,NONE",
```

Comment:
was this test even running?cause I can see you had to add NONE 4 times

Reply:
Test is running, but logic only depends on the first shared_dir parameter, so other FSx params do not matter.
Added a comment in the code to explain this